stats/statsd.go

```diff
@@ -135,10 +135,31 @@ func formatStatsdTags(tags []interface{}) string {
 
 	tags = deduplicateTags(normalizeTags(tags))
 
-	var buf bytes.Buffer
+	var buf string
```

Comment:
Completely irrelevant, but it is no longer a buffer, so rather tagsString, ts, s, or t.